src/code.sh

```
@@ -0,0 +1,52 @@
+
+# download all inputs, untar plug-n-play resources, and set PATH to arriba directory
+# mark-section "Download inputs and set up initial directories and values"
+if ! tar xvzf /home/dnanexus/in/genome_lib/*.tar.gz -C /home/dnanexus/genome_lib; then
+    echo "Error: Failed to extract genome library"
+    exit 1
+fi
+cd arriba_v2.4.0 && make
+cd ..
+export PATH=$PATH:/home/dnanexus/arriba_v2.4.0
+
+bams=($(ls /home/dnanexus/input_bams/*.bam))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)